src/components/ScrollOnRender/ScrollOnRender.tsx

```diff
@@ -1,14 +1,17 @@
-import type { PropsWithChildren } from "react";
-
-import { useScrollOnRender } from "hooks/useScrollOnRender";
+import { useEffect, type PropsWithChildren } from "react";
 
 type Props = {
   scrollArea?: HTMLElement | null;
 } & PropsWithChildren;
 
 const ScrollOnRender = ({ children, scrollArea }: Props) => {
-  const onRenderRef = useScrollOnRender(scrollArea);
-  return <div ref={onRenderRef}>{children}</div>;
+  useEffect(() => {
+    if (scrollArea) {
+      scrollArea.scrollIntoView();
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
Is there a reason we don't want to have `scrollArea` in the dependencies? If `scrollArea` is initially undefined and then gets defined wouldn't we want the `useEffect` to get called again?